Source/WebKit/WebProcess/WebPage/WebCookieJar.cpp

```diff
@@ -156,7 +156,7 @@ String WebCookieJar::cookies(WebCore::Document& document, const URL& url) const
     auto pageID = webFrame->page()->identifier();
 
     if (isEligibleForCache(*webFrame, document.firstPartyForCookies(), url))
-        return m_cache.cookiesForDOM(document.firstPartyForCookies(), sameSiteInfo, url, frameID, pageID, includeSecureCookies);
+        return m_cache.cookiesForDOM(document, document.firstPartyForCookies(), sameSiteInfo, url, frameID, pageID, includeSecureCookies);
```

Comment:
Maybe we can omit the second parameter and let the implementation call `document.firstPartyForCookies()` now that we pass it the `document`.